client/my-sites/checkout/upsell-nudge/professional-email-upsell/style.scss

```
@@ -15,12 +15,80 @@
 
 .professional-email-upsell__header {
 	text-align: center;
+
+	.billing-interval-toggle {
+		display: flex;
+
+		.segmented-control {
+			background-color: #f2f2f2;
+			border: solid 1px var( --studio-gray-5 );
```

Nit: we should stick to one order for the border attribute, which is generally (width style color) in Calypso.
```suggestion
			border: 1px solid var( --studio-gray-5 );
```